.github/workflows/tests.yml

```
@@ -55,7 +55,7 @@ jobs:
       - name: Upload coverage to Codacy
         # Only upload on push to main or PRs (not on manual triggers)
         if: github.event_name != 'workflow_dispatch'
-        uses: codacy/codacy-coverage-reporter-action@89d6c85cfafaec52c72b6c5e8b2878d33104c699 # v1.3.0
+        uses: codacy/codacy-coverage-reporter-action@a0bb89b88b5d69a6fec4a6b33bb3bbbae47b32c4 # v1.4.1
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)